setup.py

```diff
@@ -83,6 +83,7 @@ def read(fname):
         "Programming Language :: Python :: 3.9",
         "Programming Language :: Python :: 3.10",
         "Programming Language :: Python :: 3.11",
+        "Programming Language :: Python :: 3.12",
```

Comment:
Should we advertise 3.12 here? What if 3.12 stable actually breaks something we can't fix right away?

I don't mind cause I don't think it'll happen and this tag is essentially informative. Just a theoretical issue.

Reply:
I think it's fine, especially during release candidate phase when the ABI is frozen:

> There will be **no ABI changes** from this point forward in the 3.12 series. The intent is for the final release of 3.12.0, scheduled for Monday, 2023-10-02, to be identical to this release candidate. **This is the last chance to find critical problems in Python 3.12.**

https://discuss.python.org/t/python-3-12-0rc2-final-release-candidate-released/33105?u=hugovk